src/test/java/nl/knaw/dans/layerstore/LayerDatabaseAddDirectoryTest.java

```diff
@@ -163,6 +165,42 @@ public void should_add_item_records_even_if_directories_already_exist_in_another
             );
     }
 
+    @Test
```

Comment:
Why does this generate a OLE? Isn't the database empty at the start of the test?

Reply:
Yes, it is empty. As the name of the test says: the `generatedId` should be in the DB. Replaced the expectations with a less obscure part of the message: "actual row count: 0; expected: 1"